examples/java/src/main/java/org/apache/beam/examples/common/ExampleUtils.java

```diff
@@ -66,6 +66,8 @@
 
   private static final int SC_NOT_FOUND = 404;
 
+  public static final String TOKENIZER_PATTERN = "[^\\p{L}]+";
```

Comment:
Could you explain what does this do, how does it work with a short comment here? This would be helpful for future readers of this code.